### PR TITLE
Firebase InAppMessaging KTX extensions docs

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

**Comment by rlazo** [comment]
@MeghaB I've added entries for both "InAppMessaging" and "InAppMessagingDisplay" KTX extensions in `docs/README.md`. The main `README.md` **does not** mention "InAppMessaging" so I haven't included the KTX extension either. Please confirm that's correct. Thanks!

File: .opensource/project.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
         "docs/ktx/common.md": "Common KTX",
         "docs/ktx/firestore.md": "Firestore KTX",
         "docs/ktx/functions.md": "Functions KTX",
+        "docs/ktx/fiam.md": "InApp Messaging KTX",
```
Comment (rosariopfernandes) [inline .opensource/project.json]:
```suggestion
        "docs/ktx/fiam.md": "In-App Messaging KTX",
```

File: .opensource/project.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
...
+        "docs/ktx/fiamd.md": "InApp Messaging Display KTX",
```
Comment (rosariopfernandes) [inline .opensource/project.json]:
```suggestion
        "docs/ktx/fiamd.md": "In-App Messaging Display KTX",
```

File: docs/ktx/fiam.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# InAppMessaging Kotlin Extensions
```
Comment (rosariopfernandes) [inline docs/ktx/fiam.md]:
```suggestion
# In-App Messaging Kotlin Extensions
```

File: docs/ktx/fiam.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Getting Started
+
+To use the Firebase InAppMessaging Android SDK with Kotlin Extensions, add the following
```
Comment (rosariopfernandes) [inline docs/ktx/fiam.md]:
```suggestion
To use the Firebase In-App Messaging Android SDK with Kotlin Extensions, add the following
```

File: docs/ktx/fiamd.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# InAppMessagingDisplay Kotlin Extensions
+
+## Getting Started
+
+To use the Firebase InAppMessaging Display Android SDK with Kotlin Extensions, add the following
```
Comment (rosariopfernandes) [inline docs/ktx/fiamd.md]:
```suggestion
To use the Firebase In-App Messaging Display Android SDK with Kotlin Extensions, add the following
```

File: .opensource/project.json [hunk 6]
```diff
@@ -10,6 +10,8 @@
         "docs/ktx/common.md": "Common KTX",
         "docs/ktx/firestore.md": "Firestore KTX",
         "docs/ktx/functions.md": "Functions KTX",
+        "docs/ktx/fiam.md": "In-App Messaging KTX",
+        "docs/ktx/fiamd.md": "In-App Messaging Display KTX",
```
Comment (rachelsaunders) [inline .opensource/project.json]: Edit file names, pls:
- fiam.md -> inappmessaging.md
- fiamd.md -> inappmessaging-display.md

Comment (rlazo) [reply .opensource/project.json]: Done!

**Comment by rlazo** [comment]
> Pls address rlazo@'s comment about the missing mention of In-App Messaging (not display) in the main README (firebase-android-sdk/docs/README.md). Thx!

Done, PTAL @rachelsaunders  !

**Pull request merged**